src/domains/user/service.ts

```diff
@@ -1,5 +1,18 @@
 import { api } from "@/services/api";
+import { CreateUserData } from "../auth";
 
 export function getMe() {
-  return api.get("me");
+  return api.get("auth/me");
+}
```

Comment:
Try to return only data from services if possible. Return the whole response only if necessary
```ts
export function getMe() {
  return (await api.get("auth/me")).data;
}
```